packages/common/driver-definitions/src/storage.ts

```diff
@@ -385,6 +385,32 @@ export interface IDocumentServiceFactory {
 	): Promise<IDocumentService>;
 }
 
+export interface ISocketStorageDiscoveryParts {
+	/**
+	 * Niche interface to allow odsp-driver to expose specific pieces necessary for /broadcast-signal
+	 * endpoint. Please do not use this for anything else. We will phase this out once we know how
+	 * the interface for a more uniform interface across multiple drivers looks.
+	 */
+
+	// SPO gives us runtimeTenantId, we remap it to tenantId
+	// See getSocketStorageDiscovery
+	runtimeTenantId?: string;
```

Comment:
Is this permanently optional or temporary as a staging measure?  Both here and ISocketStorageDiscovery would benefit from docs on the members of the interfaces.

Reply:
@jatgarg do you know the answer to this one? I just copy pasted from ISocketStorageDiscovery.